pentaho-gwt-widgets/src/org/pentaho/mantle/client/events/EventBusUtil.java

```diff
@@ -21,11 +21,12 @@
 import com.google.gwt.core.client.JavaScriptObject;
 import com.google.gwt.event.shared.EventBus;
 import com.google.gwt.event.shared.SimpleEventBus;
+import com.google.gwt.event.shared.HandlerRegistration;
 
 public interface EventBusUtil {
   public static final EventBus EVENT_BUS = GWT.create( SimpleEventBus.class );
 
-  public void addHandler( String eventType, JavaScriptObject handler );
+  HandlerRegistration addHandler( String eventType, JavaScriptObject handler );
 
```

Comment:
Rebase the EWOK branch against master and you get that change in - we needed to add it during the merge as the user-console changes did not compile without it.